### PR TITLE
exodus.py: Some minor refactors and tests.

### PR DESCRIPTION
**Pull request by mvlopri**
Refactor the ex_obj_to_inq and ex_entity_type_to_objType
dictionary lookups and add unit tests for them.
Also add some tests for the functionality of assemblies
through the python interface.

### CODE REVIEW

File: packages/seacas/scripts/exodus3.in.py [hunk 1]
```diff
@@ -2345,7 +2349,7 @@ def put_elem_blk_names(self, names):
 
         >>> exo.put_elem_blk_names(elem_blk_names)
 
-        Parameters
+        Parametersf
```
Comment (gsjaardema) [inline packages/seacas/scripts/exodus3.in.py:2352]: Minor: Spelling / extra f on end

File: packages/seacas/scripts/exodus3.in.py [hunk 2]
```diff
@@ -70,10 +70,10 @@
...
 EXODUS_PY_COPYRIGHT_AND_LICENSE = __doc__
 
-EXODUS_PY_VERSION = "1.20.1 (seacas-py3)"
+EXODUS_PY_VERSION = "1.20.2 (seacas-py3)"
 
 EXODUS_PY_COPYRIGHT = """
-You are using exodus.py v 1.20.1 (seacas-py3), a python wrapper of some of the exodus library.
+You are using exodus.py v 1.20.2 (seacas-py3), a python wrapper of some of the exodus library.
 
 Copyright (c) 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020 National Technology &
```
Comment (gsjaardema) [inline packages/seacas/scripts/exodus3.in.py:78]: Minor -- We should add 2021 to copyright list of years.

**Review by gsjaardema — APPROVED** [review]
Looks good.  A couple very minor comments.

File: packages/seacas/scripts/exodus3.in.py (hunk 1 above)
Comment (mvlopri) [reply packages/seacas/scripts/exodus3.in.py]: Nice catch. I will send up a different commit that has those fixes.

Comment (mvlopri) [reply packages/seacas/scripts/exodus3.in.py]: I think these should be resolved now.

**Comment by gsjaardema** [comment]
The PR still only has that original commit although i see that your ddc3df9 has the changes.  Somehow it didn't get added to this PR.

**Comment by gsjaardema** [comment]
nevermind.  Not sure what I was looking at, but everything looks good.

**Review by gsjaardema — APPROVED** [review]
Looks good.  Thanks.

**Pull request merged**